src/l10n.json

```diff
@@ -18,7 +18,6 @@
     "general.conferences": "Conferences",
     "general.copyright": "Scratch is a project of the Lifelong Kindergarten Group at the MIT Media Lab",
     "general.copyrightDraft": "Scratch is a project of the Scratch Foundation in collaboration with the Lifelong Kindergarten Group at the MIT Media Lab.",
-    "general.country": "Country",
```

Comment:
Is this still used here? https://github.com/LLK/scratch-www/blob/46de5a23e17c5d3a5226e29e54f09174c3264706/src/components/registration/steps.jsx#L556

Reply:
You're right! I put it back.